scripts/clean_db_and_setup_with_fake_users.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+# To run script in Docker call for example "delete_db_and_setup.sh 'docker-compose exec backend'"
+EXEC_ENV="$*"
+DJANGO_SUPERUSER_PASSWORD="admin"
+
+$EXEC_ENV echo "DO NOT USE THIS SCRIPT IN PRODUCTION"
```

Comment:
Why don't we leave it general enough for production and development? I have actively decided against including the `createfakeusers` etc.

Reply:
Well then just delete all my commits. I have an alias for this, but then you could just make the script docker only. If you need to enter this many things and also create fake users manually, I won't use it for development, and I suspect other wont aswell.

Reply:
Merged both scripts